Pokemon/Pokemon/PokemonList.h

```diff
@@ -0,0 +1,23 @@
+//
+//  PokemonList.h
+//  Pokemon
+//
+//  Created by Brian Blanco on 7/31/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface PokemonList : NSObject
```

Comment:
It looks like this class represents a single Pokemon, so it should probably just be called `Pokemon`.